contracts/programs/access-controller/src/lib.rs

```diff
@@ -19,8 +19,8 @@ pub const MAX_ADDRS: usize = 64;
 
 #[zero_copy]
 pub struct AccessList {
-    xs: [Pubkey; 64], // sadly we can't use const https://github.com/project-serum/anchor/issues/632
     len: u64,
+    xs: [Pubkey; MAX_ADDRS],
```

Comment:
Why the order switch here?

I've noticed you usually put data first, and the data length last.

Reply:
https://github.com/smartcontractkit/chainlink-solana/runs/4786341439?check_suite_focus=true